jobs/env/ci-kubernetes-e2e-gci-gce-scalability-stable.env

```diff
@@ -2,6 +2,11 @@
 
 # Bump COS version to avoid kernel panics soflookups
 # after last security patch: pr.k8s.io/61047
-# TODO: This is needed only in releases 1.8 and 1.9.
+# TODO: This is needed only in release 1.9.
 # Remove once these releases are no longer supported.
 KUBE_GCI_VERSION=cos-stable-63-10032-71-0
+
+# Avoid overwriting qps limits to 100 for controller manager
+# and scheduler - these doesn't work in 1.9 and 1.10 releases.
+CONTROLLER_MANAGER_TEST_ARGS=--profiling
+SCHEDULER_TEST_ARGS=--profiling
```

Comment:
Could you not change this for scheduler, but only for c-m (I suspect the latter is unrelated wrt DELETE pods latency).